### ISSUE TITLE
Model on Zenodo not the same as in EvaluateNER notebook?

### ISSUE THREAD

**Issue by twielfaert** [nlesc-sherlock/spaCy-dutch#2]
Is it possible that the Dutch models shared [here](https://zenodo.org/record/202662#.WUD452jyhhE) is actually not the one which results we get to see in the EvaluateNER notebook?

Reproducing the notebook didn't work as expected as processing the toy sentence produces the following result (`Piet PART  O
leest SCONJ  O
een CONJ  O
boek INTJ  O
. PRON  O`) and the accuracy for the CONLL test set are equally dramatic.

**Comment by jvdzwaan** [comment]
If you want to use the Dutch models, you have to set them up separately. If you didn't set them up (see instructions under 'Add Dutch language data folder' in the readme), you are not using them.

Also, the way languages are set up in spaCy has changed since we created the Dutch models, so you'll need to take into account those changes if you want to use the Dutch data. That could also be a reason why it doesn't work.

**Comment by twielfaert** [comment]
I did add Dutch and if I understand the structure of spaCy correctly, NER output will not be formatted correctly if the model isn't present.

It is definitely so though, that spaCy is evolving quickly (v2 is now in alpha phase), which makes it quite tricky to add new languages.

Could you elaborate on what has changed in spaCy since you trained the Dutch models?

**Comment by jvdzwaan** [comment]
> Could you elaborate on what has changed in spaCy since you trained the Dutch models?

No, sorry, I haven't looked at the changes at all. Just picked up that things have changed...

**Comment by twielfaert** [comment]
I was able to reproduce the results using the "dutch" branch of spaCy, so it seems indeed that the current version of spaCy has indeed changed too much to reuse language models trained through older versions.

**Comment by jvdzwaan** [comment]
The data could also just be put at a different location...
